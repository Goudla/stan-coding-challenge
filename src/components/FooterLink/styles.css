.footer-link {
  color: #bdbdbd;
  font-size: 14px;
  font-weight: 600;
  text-decoration: none;
}

.footer-link:not(:last-child):after {
  border-right: 1px solid #999;
  content: '';
  margin: 0 10px 0 10px;
}

.footer-link:hover,
.footer-link:focus,
.footer-link:active {
  color: #09f;
}
