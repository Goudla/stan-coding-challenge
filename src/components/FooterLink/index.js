import * as React from 'react';

// Styles
import './styles.css';

type Props = {
  children: React.Node
};

class FooterLink extends React.PureComponent<Props> {
  render() {
    const { children } = this.props;

    return (
      <a href="https://www.stan.com.au/" className="footer-link">
        {children}
      </a>
    );
  }
}

export default FooterLink;
