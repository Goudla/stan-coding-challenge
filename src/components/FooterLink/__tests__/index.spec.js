import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import FooterLink from '../';

describe('<FooterLink />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<FooterLink>Test</FooterLink>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
