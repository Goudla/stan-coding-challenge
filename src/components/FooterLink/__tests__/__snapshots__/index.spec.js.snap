// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterLink /> Should render 1`] = `
<a
  className="footer-link"
  href="https://www.stan.com.au/"
>
  Test
</a>
`;
